### PR TITLE
Upgarde to Hawkular Parent 41 and compatible components  * Newest Metrics srcdep

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jshaughn and/or @lucasponce could you please review and eventually merge?

**Comment by jshaughn** [comment]
travis keeps failing, not sure if it's travis or the change...

**Comment by lucasponce** [comment]
Travis is failing in the srcdeps step:
[INFO] srcdeps-maven-plugin dependency missing in local repository: /home/travis/.m2/repository/org/hawkular/metrics/hawkular-metrics-core-service/0.15.0.Final-SRC-revision-5a9b432eb13fa4525df8198ff3949a4ad1d9e49a/hawkular-metrics-core-service-0.15.0.Final-SRC-revision-5a9b432eb13fa4525df8198ff3949a4ad1d9e49a.jar
[INFO] srcdeps-maven-plugin [JGitScm] is checking out [metrics] version [0.15.0.Final-SRC-revision-5a9b432eb13fa4525df8198ff3949a4ad1d9e49a] from URL [0] [scm:git:https://github.com/hawkular/hawkular-metrics.git]
[INFO] srcdeps-maven-plugin is setting [metrics] version [0.15.0.Final-SRC-revision-5a9b432eb13fa4525df8198ff3949a4ad1d9e49a]
[INFO] Executing: /bin/sh -c cd '/home/travis/.m2/repository/../dependency-sources/metrics' && '/usr/local/maven/bin/mvn' '-s' '/tmp/release-settings7828576083261025075.xml' '-f' 'pom.xml' 'versions:set' '--no-plugin-updates' '--batch-mode' '-q' '-DnewVersion=0.15.0.Final-SRC-revision-5a9b432eb13fa4525df8198ff3949a4ad1d9e49a' '-DgenerateBackupPoms=false'
    Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=192m; support was removed in 8.0
[INFO] srcdeps-maven-plugin is building [metrics] version [0.15.0.Final-SRC-revision-5a9b432eb13fa4525df8198ff3949a4ad1d9e49a]
[INFO] Executing: /bin/sh -c cd '/home/travis/.m2/repository/../dependency-sources/metrics' && '/usr/local/maven/bin/mvn' '-s' '/tmp/release-settings3496061095398640924.xml' '-f' 'pom.xml' 'clean' 'install' '--no-plugin-updates' '--batch-mode' '-q' '-Dcheckstyle.skip=true' '-Dlicense.skip=true' '-Drevapi.skip=true' '-Dsrcdeps.forwardProperties=' '-Dsrcdeps.quiet=true' '-Dsrcdeps.skipTests=true' '-DskipTests=true'
    Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=192m; support was removed in 8.0
No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
The build has been terminated

**Comment by ppalaga** [comment]
92bb820 replaces the previous version. I am curious if it is going to pass on Travis. I have seen one random failure locally. The main changes:

* Swagger related stuff managed in parent
* Embedded Cassandra now packaged as war

File: hawkular-alerts-engine/src/test/java/org/hawkular/commons/cassandra/EmbeddedCassandra.java [hunk 1]
```diff
@@ -1,43 +0,0 @@
-/*
```
Comment (lucasponce) [inline hawkular-alerts-engine/src/test/java/org/hawkular/commons/cassandra/EmbeddedCassandra.java]: Why this file has been dropped ?
This was placed here for some profiles where we don't use a cassandra from a .war.
Is that being tested ?

Comment (ppalaga) [inline hawkular-alerts-engine/src/test/java/org/hawkular/commons/cassandra/EmbeddedCassandra.java]: My understanding was that `EmbeddedCassandra` was just wrapping the `EmbeddedCassandraService` from Commons to make the `stop()` method public. I refactored the code using `EmbeddedCassandra` to use `EmbeddedCassandraService` from commons directly. Have I missed some occurrences?

**Comment by ppalaga** [comment]
The build of 907a0ed failed on the same location where I have seen a random failure locally as well:

```
Failed tests: 
  ActionsITest.actionByStatusTest:413 expected:<5> but was:<4>
```

Restarting Travis.

**Comment by ppalaga** [comment]
539c73a tries to solve the previously seen failure in `ActionsITest.actionByStatusTest:413 expected:<5> but was:<4>`

**Comment by lucasponce** [comment]
Travis seems ok.
It seems that the changes on the C* side could make that initialization take more time and some times we might need additional waits in the itest.
Anyway, good to merge from me.

**Pull request merged**